### PR TITLE
Enabling NFS PV reattachment

### PR DESCRIPTION
**Pull request by almahmoud**
See[ test script](https://github.com/galaxyproject/galaxykubeman-helm/blob/nfs_pvs/galaxykubeman/scripts/test_pv_restore.sh) for more details.

TLDR!
- Deploy Galaxy

```
GALAXY_PVC_ID=$(kubectl get pvc -n namespace gxy-rls-galaxy-pvc -o=jsonpath='{.metadata.uid}')
CVMFS_PVC_ID=$(kubectl get pvc -n namespace gxy-rls-cvmfs-alien-cache-pvc -o=jsonpath='{.metadata.uid}')
```
 - Store above^
 - Destroy cluster and all resources except volume
 - Boot up new cluster
 - Install same version of chart with same values, adding the following:

```
    --set restore.persistence.nfs.galaxy.pvcID=\"$GALAXY_PVC_ID\"\
    --set restore.persistence.nfs.cvmfsCache.pvcID=\"$CVMFS_PVC_ID\"\
    --set galaxy.persistence.existingClaim=\"same-gxy-rls-galaxy-pvc\"\
    --set cvmfs.cache.alienCache.existingClaim=\"same-gxy-rls-cvmfs-alien-cache-pvc\""
```

Should have the same state it was left in with all histories intact, data usable, jobs rerunnable etc...

EDIT: For the scenario where the release was deleted but the cluster was not cleaned up, and the PVs are still existing, instead of having Leo delete the PVs, I've made their creation optional so the restore can also just attach PVCs to existing PVs. The default will be to create them, so the above will not change. For the scenario where you don't want those PVs created because they still exist, then you can add the values:
```

    --set restore.persistence.nfs.galaxy.deployVolume=false
    --set restore.persistence.nfs.cvmfsCache.deployVolume=false
```

### CODE REVIEW

**Comment by rtitle** [comment]
@Qi77Qi FYI
(Qi is working on disk reattachment on the Leo side)

Question about process -- if this is ready should we tag a new version of `galaxykubeman` for integration with the Leo? Current version is `0.7.2`, so maybe `0.8.0`?

**Review by Qi77Qi — APPROVED** [review]

**Comment by almahmoud** [comment]
I pre-packaged `0.8.0` so you can already test this change with that chart. The CI will do the automatic version bumping when this is labeled properly and merged, but figured we'd wait until it's tested and ready to go in case other tweaks are needed and to potentially roll out 21.01 at the same time.
